conanfile.py

```diff
@@ -1,46 +1,84 @@
 from conans import ConanFile, AutoToolsBuildEnvironment, tools
-import sys
-import shutil
+import os
 
 
 class LibsndfileConan(ConanFile):
     name = "libsndfile"
     version = "1.0.28"
-    license = "<Put the package license here>"
+    license = ("LGLP-2.1", "LGPL-3.0")
     author = "Martin Delille <martin@phonations.com>"
     url = "https://github.com/Phonations/conan-libsndfile"
+    homepage = "http://www.mega-nerd.com/libsndfile"
     description = "A C library for reading and writing sound files containing sampled audio data."
-    topics = ("audio")
+    exports = "LICENSE"
+    topics = ("audio", "sound", "wav", "aiff", "sampled-sound")
     settings = "os", "compiler", "build_type", "arch"
-    options = {"shared": [True, False]}
-    default_options = "shared=False"
-    generators = "cmake"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
```

Comment:
What is the `fPIC` option? I don't understand what it change to add it because there is no reference to it in the recipe (unlike `shared`).

Reply:
[fPIC](https://gcc.gnu.org/onlinedocs/gcc/Code-Gen-Options.html) means Position Independent Code. It's automatically managed by when that option is declared: https://docs.conan.io/en/latest/reference/build_helpers/autotools.html#fpic